internal/cmd/commands/accountscmd/accounts.gen.go

```diff
@@ -311,6 +312,19 @@ func (c *Command) Run(args []string) int {
 	return base.CommandSuccess
 }
 
+func (c *Command) checkFuncError(err error) int {
+	if err != nil {
```

Comment:
nit: We generally try and avoid extra indentations be rather return early, here we could:
```
if err == nil {
   return 0
}
...
```